docs/chat.md

```diff
@@ -393,3 +393,5 @@ See [OCP\RichObjectStrings\Definitions](https://github.com/nextcloud/server/blob
 * `message_expiration_disabled` - {actor} disabled message expiration
 * `breakout_rooms_started` - {actor} started breakout rooms
 * `breakout_rooms_stopped` - {actor} stopped breakout rooms
+* `set_room_avatar` - {actor} set the conversation avatar
+* `delete_room_avatar` - {actor} deleted the conversation avatar
```

Comment:
current pattern is `<object>_<verb>` and vers are in past tense.
Also would use "removed" to follow the wording of description changes:

```suggestion
* `avatar_set` - {actor} set the conversation avatar
* `avatar_removed` - {actor} removed the conversation avatar
```